### ISSUE TITLE
Upload of data

### ISSUE THREAD

**Issue by phildonaldson1968** [tensorflow/playground#111]
Fantastic visualisation of Neural Nets - would be superb if a user could upload data with the same structure as used in the examples.  Thanks anyway, very interesting and super tool!

**Comment by chaitanya11** [comment]
+1

**Comment by feature-engineer** [comment]
In case of classification, the uploaded data must be symmetric about the center of the distribution for the training to work well, since the loss function is MSE.

MSE - Mean Squared Error, is usually employed for regression and not for classification problems and only works for the examples thanks to their symmetry, since this loss is very sensitive to outliers.
